charts/jaeger/templates/cassandra-schema.yaml

```diff
@@ -58,6 +58,7 @@ spec:
         {{- end }}
         resources:
           {{- toYaml .Values.schema.resources | nindent 10 }}
+        {{- if or .Values.schema.extraConfigmapMounts .Values.storage.cassandra.tls.enabled }}
```

Comment:
The main binary has the ability to create schema, we don't need an external job anymore. Can we drop this template?